prov/efa/src/efa_prov_info.c

```diff
@@ -75,7 +75,7 @@ const struct fi_domain_attr efa_domain_attr = {
 	.resource_mgmt		= FI_RM_DISABLED,
 	.mr_mode		= OFI_MR_BASIC_MAP | FI_MR_LOCAL | OFI_MR_BASIC,
 	.mr_key_size		= sizeof_field(struct ibv_sge, lkey),
-	.cq_data_size		= 0,
+	.cq_data_size		= 4,
```

Comment:
This should be a general macro as `EFA_CQ_DATA_SIZE` (, and we should remove this https://github.com/ofiwg/libfabric/blob/main/prov/efa/src/efa_prov_info.c#L557-L559 and https://github.com/ofiwg/libfabric/blob/main/prov/efa/src/rdm/efa_rdm_cq.h#L18-L21 after having it by default